swagger.yaml

```diff
@@ -477,6 +477,7 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/Error"
+  /v1/vacationsRequests/{id}/{personId}:
```

Comment:
So previously a vacation request was deleted by the vacation request id, now it also requires the personId. I assume there is a reason for this?